internal/stacks/stackruntime/internal/stackeval/planning_test.go

```diff
@@ -920,3 +920,28 @@ func TestPlanning_NoWorkspaceNameRef(t *testing.T) {
 		}
 	})
 }
+
+func TestPlanning_Locals(t *testing.T) {
+	// This test verifies that a reference to terraform.workspace is treated
```

Comment:
Is this comment accurate for this test? It seems to be talking about input variables, so I'm wondering if this comment belonged to another test that you derived this one from.